packages/backend/test/core/tx-count/BlockTxCountUpdater.test.ts

```diff
@@ -53,7 +53,7 @@ describe(BlockTxCountUpdater.name, () => {
       })
       const txCountRepository = mock<TxCountRepository>({
         findLatestByProject: async () => fakeTxCount({ blockNumber: 4 }),
-        getMissingRangeByProject: async () => [[1, 3]],
+        getMissingRangesByProject: async () => [[1, 3]],
```

Comment:
This is a value that this function will never return

Reply:
Outdated